spec/serializers/api/admin/order_cycle_serializer_spec.rb

```diff
@@ -18,11 +18,15 @@
   end
 
   it "serializes the order cycle with editable_variants_for_incoming_exchanges" do
-    expect(serializer.editable_variants_for_incoming_exchanges.to_json).to include order_cycle.variants.first.id.to_s
-    expect(serializer.editable_variants_for_incoming_exchanges.to_json).to_not include order_cycle.distributors.first.id.to_s
+    expect(from_json(serializer.editable_variants_for_incoming_exchanges).values.sum).to include order_cycle.variants.first.id
```

Comment:
It took me a minute to work out why you're using `sum` here. I guess this is a quick way to test both that the ID is in there, and that it is also the only one. 
Hmm but you're still using `include` rather than `eq`, so did I get that right?

Reply:
Maybe we need to write that better then. The values are arrays of ids. Array plus array is a new array. And then I test if the id is in that one array of variant ids.

Reply:
I missed that somehow. I believe he is using sum to flatten the array as `values` here would return something like `[[1,2], [3,4]]` 
 I think it would be clearer to use `flatten` like this :
```ruby
expect(from_json(serializer.editable_variants_for_incoming_exchanges).values.flatten).to include order_cycle.variants.first.id
```

Reply:
Thanks, not being familiar with the data structure, I find `flatten` nice and clear. 
But giving the array a variable name also helps a lot. 👍

Reply:
I agree that `flatten` would have been better. But I don't want to add another commit just for that. Or do I? Why not.